.github/actions/lint/lint_links.py

```diff
@@ -30,12 +39,63 @@ def main():
 
 
 ## HELPER FUNCTIONS
-def get_markdown_files(root_dir):
+def parseInputArguments():
+    """
+    Parsing arguments for the root directory and config file.
+
+    Returns:
+        ArgumentParser: An argument parser object used for accessing command line arguments.
```

Comment:
I'm nit picking here: it actually returns a [`Namespace`](https://docs.python.org/3/library/argparse.html#the-namespace-object) object.